core/appmodule/v2/module.go

```diff
@@ -47,6 +49,10 @@ type HasEndBlocker interface {
 	EndBlock(context.Context) error
 }
 
+type HasABCIEndBlock interface {
```

Comment:
do we want that to be a v2 thing that third-party modules can implement?
I would just do the interface check maybe in runtime/v2 for backward compatibility but not add it to core

Reply:
That sounds reasonable to me.